sparkly/catalog.py

```diff
@@ -26,6 +27,57 @@ def __init__(self, spark):
         """
         self._spark = spark
 
+    def create_external_table(self, table_name, path=None, source=None, schema=None, **options):
```

Comment:
In 2.1, `createExternalTable` is deprecated in favor of `createTable`. I wonder if we should make this `create_table`, optionally with a synonym for `create_external_table`